drivers/hmis/app/graphql/types/hmis_schema/has_audit_history.rb

```diff
@@ -29,8 +29,20 @@ def audit_history_field(name = :audit_history, description = nil, association_na
 
           # Override "resolve_audit_history" to override default scope behavior
           define_method(:resolve_audit_history) do
-            # Unscope to remove default order, otherwise it will conflict
-            object.send(association_name).where.not(object_changes: nil, event: 'update').unscope(:order).order(created_at: :desc)
+            address_ids = object.addresses.with_deleted.pluck(:id)
+            name_ids = object.names.with_deleted.pluck(:id)
+            contact_ids = object.contact_points.with_deleted.pluck(:id)
```

Comment:
These are only going to work on client objects.  We we want to keep this more agnostic, or just fix it up when the next scenario arises?

Reply:
Oh good point, I'll move this onto the Client type is an override to this function.